internal/cmd/forwarder-generator/main.go

```diff
@@ -1,21 +1,28 @@
 package main
 
 import (
+	"bufio"
+	"flag"
 	"fmt"
 	"io/ioutil"
 	"os"
 
-	yaml "gopkg.in/yaml.v2"
+	"github.com/spf13/pflag"
 
-	logging "github.com/openshift/cluster-logging-operator/pkg/apis/logging/v1"
-	"github.com/openshift/cluster-logging-operator/pkg/generators/forwarding"
-	"github.com/openshift/cluster-logging-operator/pkg/k8shandler"
+	"github.com/openshift/cluster-logging-operator/internal/pkg/generator/forwarder"
 	"github.com/openshift/cluster-logging-operator/pkg/logger"
 	"github.com/sirupsen/logrus"
 )
 
 // HACK - This command is for development use only
 func main() {
+
+	yamlFile := flag.String("file", "", "ClusterLogForwarder yaml file. - for stdin")
+	includeDefaultLogStore := flag.Bool("include-default-store", true, "Include the default storage when generating the config")
```

Comment:
Maybe we dont.  It was added because the code this calls into has a boolean flag that represents this.  I did not follow the call but AFAIK this is the only way you get the default store.